skrub/_tabular_learner.py

```diff
@@ -104,7 +114,7 @@ def tabular_learner(estimator, n_jobs=None):
     ...         "fasting_glucose": [35, 140, 44, 137],
     ...     }
     ... )
-    >>> y = [False, True, False, True]
+    >>> y = [0, 1, 0, 1]
```

Comment:
I would not encourage to not use integral classification values when this is possible